src/main/java/com/puppycrawl/tools/checkstyle/checks/annotation/AnnotationLocationCheck.java

```diff
@@ -31,13 +31,18 @@
  * Check location of annotation on language elements.
  * By default, Check enforce to locate annotations immediately after
  * documentation block and before target element, annotation should be located
- * on separate line from target element.
+ * on separate line from target element. Elements that cannot have JavaDoc comments
+ * like local variables are not in the scope of this check even though a token type like
+ * {@code VARIABLE_DEF} would match them. This check also verifies that the annotations
```

Comment:
> This check also verifies 

It seems this sentence should before the other one as it continues on with the previous sentence before that saying what the check does.

> Elements that cannot have JavaDoc comments

This sentence sounds more like it should be in an attention like the line below, maybe by itself.

Reply:
changed